Documentation/BuildInstructionsWindows.md

```diff
@@ -51,3 +51,13 @@ following command in an elevated PowerShell session: \
 You may have to reboot after enabling the WHPX feature.
 
 Afterwards you can start the VM with `Meta/serenity.sh run` as usual.
+
+## Note on Microsoft Store Ubuntu
+
+The Microsoft Store does not have the latest versions of Ubuntu.
```

Comment:
Actually it does
[https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW)